cybershell/validation_framework.py

```
@@ -0,0 +1,973 @@
+        confidences = [e.confidence for e in evidence]
+        
+        # High variance in confidence indicates inconsistency
+        if np.std(confidences) > 0.3:
+            return True
+        
+        # Check reproducibility inconsistency
+        reproducible_count = sum(1 for e in evidence if e.reproducible)
+        if 0 < reproducible_count < len(evidence):
+            return True
+        
+        return False
```

Ruff (0.12.2)

Return the condition `0 < reproducible_count < len(evidence)` directly

Replace with `return 0 < reproducible_count < len(evidence)`

(SIM103)